boa_engine/src/vm/opcode/mod.rs

```diff
@@ -126,32 +126,44 @@ macro_rules! generate_impl {
                 unsafe { std::mem::transmute(value) }
             }
 
+            const STR_MAP: [&'static str; 256] = {
+                let mut map = [""; 256];
+                $(
+                    map[Self::$Variant as usize] = $Variant::NAME;
+                )*
+                map
+            };
```

Comment:
I think this could be expressed as a static array of static strings, which is easier to maintain. Something like:

```
const STR_MAP: &[&str] = &[
    (<$Variant as Operation>::NAME,)*
];
```
(Not typechecked, so it may throw errors at the start)
Also applies to the other arrays.

Reply:
Thanks! I was just confused if the order is preserved in the macro. It seems it is.